python/lsst/meas/extensions/gaap/_gaap.py

```diff
@@ -78,46 +81,54 @@ def _isOdd(x: int) -> bool:  # noqa: N805
     sigmas = pexConfig.ListField(
         dtype=float,
         default=[4.0, 5.0],
-        doc="List of sigmas (in pixels) of circular Gaussian apertures to apply on "
+        doc=("List of sigmas (in pixels) of circular Gaussian apertures to apply on "
             "pre-seeing galaxy images. These should be somewhat larger than the PSF "
             "(determined by ``scalingFactors``) to avoid measurement failures.")
+    )
```

Comment:
```suggestion
        doc=("List of sigmas (in pixels) of circular Gaussian apertures to apply on "
             "pre-seeing galaxy images. These should be somewhat larger than the PSF "
             "(determined by ``scalingFactors``) to avoid measurement failures.")
    )
```
i.e. there's an extra whitespace at the beggining of ll. 84-85, so the double quotes are vertically aligned with those of l. 83 (instead of being aligned with the opening bracket).

That's only if you decide to keep the extra parenthesis though

Reply:
Ah, sorry, I was somehow blind to this change. I've decided to drop the parenthesis though.